### PR TITLE
Syndicate Bundles and Uplink Items

### PR DESCRIPTION
**Pull request by Kierany9**
### Unchanged feature ports
- Re-adds Hippie's Syndicate Bundles (and adds a missing return).
- Re-adds Holoparasites for 20 TC.
- Reduces Hacked AI modules from 14 to 10 TC.
- Reduces Syndicate Bombs from 11 to 10 TC.
- Reduces Syndicate Minibombs from 6 to 5 TC.
- Increases Power Sinks from 6 to 8 TC.
- Increases Thermals from 4 to 6 TC.
- Microbomb Implants can now be purchased outside of Nuclear Emergency.

### Changed feature ports
- Re-adds Mindslave Implants. They are now permanent and cost 10 TC instead of 9.
- Does not re-add Greater Mindslave Implants.
- Reduces Mulligan/Randomizer syringes from 4  to 2 TC, as opposed to Hippie's 1 TC.
- Reduces Detomatrix Cartridges from 6 to 4 TC, as opposed to Hippie's 2 TC.

### Misc stuff
- All bundles that had Greater Mindslave Implants now use regular Mindslave Implants.
- The Dark Lord bundle has had its Mindslave Implant removed.
- The Ninja Bundle has been commented out because a lot of its items were coded like ass and I don't particularly want to add new features or balancing in the rebase.
- Snake Bundle's CQC gloves have been replaced with Krav Maga gloves for the time being.

### CODE REVIEW

File: code/game/objects/items/weapons/implants/implant_mindslave.dm [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+			return 0
+		if(isloyal(target))
+			target <<"<span class='danger'>Your loyalty implant rejects [user]'s mindslave!</span>"
+			user <<"<span class='danger'>[target] somehow rejects the mindslave implant!</span>"
+			return 0
+		if(..())
+			target << "<span class='userdanger'>You feel a surge of loyalty towards [user].</span>"
+			target << "<span class='userdanger'>You have been mindslaved by [user]! Until the implant expires, they are your master. You MUST obey any command given to you and CANNOT harm your master unless they order you to do so.</span>"
+			if(time_enslaved)
+				timerid = addtimer(src,"remove_mindslave",time)
```
Comment (Carbonhell) [inline code/game/objects/items/weapons/implants/implant_mindslave.dm]: this doesn't make this permanent

**Comment by Kierany9** [comment]
- Reduces Mulligan/Randomizer syringes from 4  to 2 TC, as opposed to Hippie's 1 TC.

**Comment by Kierany9** [comment]
- Increases Power Sinks from 6 to 8 TC.
- Increases Thermals from 4 to 6 TC.
- Reduces Detomatrix Cartridges from 6 to 4 TC, as opposed to Hippie's 2 TC.

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 2]
```diff
@@ -125,11 +125,11 @@
...
 
 		if("boom")
-			for(var/i = 0, i < 20, i++)
+			for(var/i in 1 to 20)
 				new /obj/item/weapon/c4(src)
-			for(var/i = 0, i < 4, i++)
+			for(var/i in 1 to 4)
 				new /obj/item/weapon/grenade/syndieminibomb(src)
-			for(var/i = 0, i < 1, i++)
+			for(var/i in 1 to 1)
```
Comment (Carbonhell) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: what the fuck is this,wew?

File: code/game/objects/items/weapons/implants/implant_mindslave.dm [hunk 3]
```diff
@@ -0,0 +1,75 @@
+/obj/item/weapon/implant/mindslave/
```
Comment (Carbonhell) [inline code/game/objects/items/weapons/implants/implant_mindslave.dm]: ending slash is uglyyy

**Pull request merged**